packages/plugin-ext-vscode/src/node/scanner-vscode.ts

```diff
@@ -79,7 +81,7 @@ export class VsCodePluginScanner extends TheiaPluginScanner implements PluginSca
             startMethod: 'activate',
             stopMethod: 'deactivate',
 
-            backendInitPath: __dirname + '/plugin-vscode-init.js'
+            backendInitPath: path.join(__dirname, 'plugin-vscode-init'),
```

Comment:
Nitpick: redundant `,` at EOL. Afaik we don't enforce a member delimiter style, so feel free to ignore this.